### DISCUSSION TITLE
46838C2F-509C-08CE-5238-7B46C95F3154

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#69] [General]
# 46838C2F-509C-08CE-5238-7B46C95F3154

HD 154088, also known as Gliese 652, is a K-type main-sequence star about 59.6 light-years away in the constellation Ophiuchus. A superterran planet was first spotted around this star in 2011, but its existence was only confirmed as recent as 2021

https://celestia.mobi/addon?item=46838C2F-509C-08CE-5238-7B46C95F3154&lang=ar

**Comment by wathk-os** [comment]
Pls
